geometry_msgs/Pose[] val
float64 time
bool execute
---